README.md

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Nix
-        uses: DeterminateSystems/nix-installer-action@main
+        uses: DeterminateSystems/nix-installer-action@v1
```

Comment:
Let's add a checklist item in the PR template that says update the version if the PR is making a release.

Reply:
Guess we should add a PR template first, then... ;P